src/nl/hannahsten/texifyidea/inspections/latex/LatexFileNotFoundInspection.kt

```diff
@@ -47,7 +44,25 @@ open class LatexFileNotFoundInspection : TexifyInspectionBase() {
     override fun inspectFile(file: PsiFile, manager: InspectionManager, isOntheFly: Boolean): MutableList<ProblemDescriptor> {
         val descriptors = descriptorList()
 
-        val commands = file.commandsInFile()
+
+        val allcommands = file.commandsInFileSet() //get all comands of project
+        val commands = file.commandsInFile()  //get commands of this file
+
+        var generaloffsetdefined = ""
```

Comment:
What does this variable do? Maybe it needs a clearer name.

Use camelCase:

```suggestion
        var generalOffsetDefined = ""
```

Reply:
Yeah of course thank you.